.github/workflows/velocity.yml

```diff
@@ -0,0 +1,71 @@
+name: Velocity report
+
+on:
+  schedule:
+    # Wednesdays at 11am (after Tuesday demos)
```

Comment:
Good catch. My intent was to give a good amount of time after demos to allow for things to settle, but this comment made me realize that:
* Wednesday at 11am might be a little too heavy-handed
* Schedules are defined in UTC, so this would actually run at 4am on Wednesday

I think it makes sense to run the report at 3:30p Tuesdays PT.